client/me/action-remove/index.jsx

```diff
@@ -1,32 +1,23 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	classnames = require( 'classnames' ),
-	omit = require( 'lodash/omit' ),
-	classNames = require( 'classnames' );
-
-module.exports = React.createClass( {
+import React from 'react';
+import classNames from 'classnames';
+import omit from 'lodash/omit';
 
+export default React.createClass( {
 	displayName: 'ActionRemove',
 
-	getDefaultProps: function() {
-		return {
-			isSubmitting: false,
-			isPrimary: true
-		};
-	},
-
-	render: function() {
-		var buttonClasses = classNames( {
+	render() {
+		const buttonClasses = classNames( {
```

Comment:
I know it was this way previously, but there are two recommendations I'd make to simplify the `classNames` usage:

- If there's a property in the object always `true`, it should be moved to its own argument
- Instead of calling the function twice, you can pass as many arguments to the function as necessary

A better version would be:

```js
const buttonClasses = classNames( 'action-remove', this.props.className );
```

Which could potentially be inserted into the `className` prop directly.